pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -58,6 +59,7 @@ var basicNSTemplateTier = newNsTemplateTier("basic", "code", "dev", "stage")
 
 func TestUserSignupCreateMUROk(t *testing.T) {
 	// given
+	metrics.Reset()
```

Comment:
Resetting the gauge in the `prepareReconcile` func was problematic because the `InitializeCounter` func was called before it so I changed `metrics.Reset` to `metrics.ResetCounters` so it only resets the counters
Done in https://github.com/codeready-toolchain/host-operator/commit/9a6b4f7b0c7d0040db9c4d2a7220472360ccf2a6